ruff_lsp/server.py

```diff
@@ -95,6 +97,7 @@
 
 TOOL_MODULE = "ruff.exe" if sys.platform == "win32" else "ruff"
 TOOL_DISPLAY = "Ruff"
+VERSION_REQUIREMENT = SpecifierSet(">=0.0.291,<0.2.0")
```

Comment:
we're kinda on the intersection of semver and pep440/pep508 but packaging seems convenient